src/contexts/user.context.js

```diff
@@ -0,0 +1,38 @@
+import React, { createContext, useContext, useState } from 'react'
+import PropTypes from 'prop-types'
+import eu from '../assets/images/eu.jpeg'
+
+const UserContext = createContext()
+
+export function UserProvider({ children }) {
+  const [name, setName] = useState('Rodrigo Eloy')
+  const [eMail, setEmail] = useState('rodrigo.cavalcanti@ccc.ufcg.edu.br')
```

Comment:
Acredito que `email` poderia estar todo em minusculo.

```suggestion
  const [email, setEmail] = useState('rodrigo.cavalcanti@ccc.ufcg.edu.br')
```